docs/contributing/coding-style.md

```diff
@@ -776,6 +778,34 @@ tests := map[string]func(t *testing.T) test {
     }
     ```
 
+#### Input agreement
```

Comment:
Do you mean `input arguments`?
```suggestion
#### Input arguments
```

Reply:
No. I meant `agreement`. Is this strange?

Reply:
@kevindiu remind

Reply:
🤔 I think it is not `agreement`, but maybe it is ok.
How about this?
```suggestion
#### Inputs
```
I will let you know if I can think of another better words.